data/brands/shop/furniture.json

```diff
@@ -1422,7 +1422,7 @@
       "displayName": "XXXLutz",
       "id": "xxxlutz-7a4d3d",
       "locationSet": {
-        "include": ["at", "cz", "de", "sk"]
+        "include": ["at", "bg", "ch", "cz", "de", "hr", "hu", "pl", "ro", "rs" "se", "si", "sk"]
```

Comment:
XXXLutz is called `Aiko XXXL` in Bulgaria. It would be nice to have a `matchNames` with `Aiko`, since not all of them have been renamed.

Reply:
@arch0345 I wasn't clear enough, my bad. XXXLutz has shops in Bulgaria but they are called `Aiko XXXL`, not `XXXKutz``. Can you add a new entry for them?

Reply:
Thanks for catching my mistake - I added it without a wikidata for now as it doesn't seem to have a page yet.

Reply:
I've just created it. Q110614670